packages/lens/lens.1.0.1/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "1.2"
+name: "lens"
+version: "1.0.1"
```

Comment:
The opam metadata files in this repository typically don't include `name` and `version` fields as those values are implied from the directory structure. Not including them makes it easier to copy and compare opam files for different versions of the same package.